Assets/Scripts/CharacterStats.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace ARPGTD.CharacterStats{
+    /// CharacterStat is a class built to handle changes in character stats
+    /// The stats themselves will be stored in the Character object
+    [Serializable]
+    public class CharacterStat
+    {
```

Comment:
same here, pick one and stick with it. this line has the open bracket on a new line, but lines below it brackets are on the same line